TOC.md

```diff
@@ -286,6 +286,7 @@
     - [增量恢复](/reference/tidb-binlog/reparo.md)
     - [Kafka 自定义开发](/reference/tidb-binlog/binlog-slave-client.md)
     - [TiDB Binlog Relay Log](/reference/tidb-binlog/relay-log.md)
+    - [集群间双向同步](/reference/tidb-binlog/bi-repl.md)
```

Comment:
Is "bi-repl" a common abbreviation for "bidirectional-replication"? If not, I suggest we change the file name to "bidirectional-replication.md" because it's more understandable for search engines (better for SEO). @WangXiangUSTC 
FYI. @TomShawn

Reply:
yes, I think we can rename it to `bidirectional-replication.md`

Reply:
Ok. Please help change the file names in the docs and docs-cn repo~ @TomShawn